lib/CL/pocl_cl.h

```diff
@@ -510,6 +510,17 @@ struct pocl_device_ops {
    * NULL */
   cl_int (*reinit) (unsigned j, cl_device_id device, const char *parameters);
 
+  /**
+   * Initialize discovery mechanism in the driver to dynamically find new
+   * devices
+   * @param disco_dev_init_callback callback in runtime to init found devices
```

Comment:
```suggestion
   * \param disco_dev_init_callback callback in runtime to init found devices
```
Let's converge/standardize to \param.